content/filters/string_filters.md

```diff
@@ -0,0 +1,273 @@
+# String Filters
+
+String filters are used to manipulate outputs of the string type.
```

Comment:
Us the same convention as you did here: `Array filters are used to modify the output of arrays.`